src/commands/web/test/run.ts

```diff
@@ -107,36 +118,63 @@ export default class WebTestRun extends Command {
           urlReplacements
         )}`
       );
-    const autifyConnect = flags["autify-connect"];
-    const { configDir, userAgent } = this.config;
+    const { configDir, cacheDir, userAgent } = this.config;
     const accessToken = getOrThrow(configDir, "AUTIFY_WEB_ACCESS_TOKEN");
     const basePath = get(configDir, "AUTIFY_WEB_BASE_PATH");
     const client = new WebClient(accessToken, { basePath, userAgent });
-    const { workspaceId, resultId, capability } = await runTest(
-      client,
-      args["scenario-or-test-plan-url"],
-      {
-        option: capabilityOption,
-        name: flags.name,
-        urlReplacements,
-        autifyConnect,
-      }
-    );
-    const testResultUrl = getWebTestResultUrl(configDir, workspaceId, resultId);
-    this.log(
-      `${emoji.get(
-        "white_check_mark"
-      )} Successfully started: ${testResultUrl} (Capability is ${capability})`
-    );
-    if (flags.wait) {
-      const waitArgs = ["--timeout", flags.timeout.toString(), testResultUrl];
-      if (flags.verbose) waitArgs.push("--verbose");
-      await WebTestWait.run(waitArgs);
-    } else {
-      this.log("To wait for the test result, run the command below:");
+
+    let autifyConnectAccessPoint = flags["autify-connect"];
+    let autifyConnectClient: AutifyConnectClient | undefined;
+    if (flags["autify-connect-client"]) {
+      autifyConnectClient = await spawnClient(configDir, cacheDir, {
+        exitOnSignal: true,
+        killBeforeWaitExit: true,
+      });
+      autifyConnectAccessPoint = autifyConnectClient.accessPoint;
+      this.log(
+        `Starting Autify Connect Client for Access Point "${autifyConnectAccessPoint}"...`
+      );
+    }
```

Comment:
These should be moved inside `runTest()` because we shouldn't start the client if it's a test plan.